webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentContentViewAddModal.js

```diff
@@ -43,15 +43,15 @@ const ComponentContentViewAddModal = ({
   useEffect(() => {
     if (!versionsLoading && componentDetails) {
       const { name, versions } = componentDetails;
-      const versionMutable = versions;
+      const versionMutable = [].concat(versions).sort((a, b) => (a.id < b.id ? 1 : -1));
```

Comment:
Why sort on the frontend?

Reply:
Good point..Let me move this to the API..